pkg/k8shandler/kibana/reconciler.go

```diff
@@ -62,10 +62,18 @@ func ReconcileKibana(requestCluster *kibana.Kibana, requestClient client.Client,
 		return err
 	}
 
+	if err := clusterKibanaRequest.createOrUpdateKibanaConsoleLinks(); err != nil {
```

Comment:
@jhadvig @periklis what is the difference between what this function creates and what `KibanaConsoleExternalLogLink` provides?

Reply:
@benjaminapetersen maybe you can speak to this too?

Reply:
@ewolinetz ConsoleLink is the CR to address/add custom links to the web console's application menu. In our case this is the Link "Logging" under the section Monitoring. This info was previously passed to the console operator through a shared ConfigMap. OTOH ConsolerExternalLink is the CR to have links to external apps like kibana. Latter are used on the pod detail view.